### PR TITLE
#5-landing page feature - Iwona

### LINKED ISSUE #5: Create React Splash page
Success criteria
* Choose theme framework
* Choose theme
* Research other similar websites' front pages
* Create front page in react

### PR DESCRIPTION
**Pull request by IwonaKrysp**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Screenshot

## Related to

Fixes #5

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have carefully reviewed my own code
- [x ] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

File: server/api.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Router } from "express";
 const router = Router();
 
 router.get("/", (_, res) => {
```
Comment (mairacagri) [inline server/api.js:5]: Hey Iwona, i see that you are defining home page route(correct me if i am wrong) and i think you forgot to close( ) if you can type 
router.get('/', function (req, res) {
  res.json({ message: "world" });
})

File: client/src/App.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+.add-video-title {
+  font-weight: 300;
+  margin-top: 35px;
+}
+.btn-upload {
+  background-color: var(--green);
+  color: white;
+}
+.btn-search {
```
Comment (mairacagri) [inline client/src/App.css:70]: Really like btn styling!

File: client/src/pages/Home.js [hunk 3]
```diff
@@ -1,44 +1,81 @@
...
+									<div className="col">
+										<ButtonSearchLogo />
+									</div>
+									<div className="col">
+										<ButtonSearchIcon />
+									</div>
+								</div>
+							</div>
+
+							<div className="col-md mt-3 ">
```
Comment (mairacagri) [inline client/src/pages/Home.js:56]: You did a great job Iwona! only one suggestion here is to use card which will display title on top then  image in the middle and description at the bottom and perhabs use some border)

**Review by mairacagri — COMMENTED** [review]
Hey Iwona great job! i left few suggestions please let me know if you agree with them!

**Review by ikanengKhoza — COMMENTED** [review]
Iwona, I want to congratulate you for raising the first PR, well done. I reviewed your code to the best of my ability. You've done a great job. Don't worry about styling because I don't think it was part of the task. I like that you used functional components rather than class based components. I don't see anything that needs redoing on your work. I think we can merge your work to the main branch.

**Comment by ikanengKhoza** [comment]
merged Iwona's pull request

**Pull request merged**